scripts/release_scripts/run_macaron.sh

```diff
@@ -378,6 +388,8 @@ proxy_var_names=(
     "HTTPS_PROXY"
     "FTP_PROXY"
     "NO_PROXY"
+    "MAVEN_OPTS"
```

Comment:
I don't think we should put `MAVEN_OPTS` and `GRADLE_OPTS` to the "proxy" section as these variable are not only for proxy settings, but also other JVM flags as well.

Reply:
But we use them for proxy settings and putting them elsewhere doesn't make that usage explicit.

Reply:
See commit [ab3999b](https://github.com/oracle/macaron/pull/434/commits/ab3999b042cef51aebf4246aeb6967afd2e92a9e)